app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
```

Comment:
- [ ] As per project requirements, when a user clicks on the Sign up/Log in button, they are supposed to be redirected to the Categories page. At the moment your app leads to a page with two buttons: `Categories` and `Logout`. I understand you are trying to make the app look beautiful, but at the momemt, you are expected to follow the project requirements.
![Screen Shot 2023-02-21 at 6 25 41 AM](https://user-images.githubusercontent.com/62849872/220371764-63aae5d1-955e-4a19-a3d1-791de0cf56c4.png)
Kindly delete the route that leads to this page and redirect both sucessful sign up and login to the categories page which should be the index page.
- See project requirement below after logging in
![Screen Shot 2023-02-21 at 6 27 51 AM](https://user-images.githubusercontent.com/62849872/220372504-7f5fcbdd-67f5-41f6-a3b1-b35235542918.png)